independent-projects/bootstrap/app-model/src/test/java/io/quarkus/bootstrap/model/PlatformImportsTest.java

```diff
@@ -0,0 +1,155 @@
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import io.quarkus.bootstrap.util.IoUtils;
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+public class PlatformImportsTest {
+
+    private final List<PlatformProps> platformProps = new ArrayList<>();
+
+    @AfterEach
+    public void cleanUp() {
+        for (PlatformProps p : platformProps) {
+            p.delete();
+        }
+    }
+
+    @Test
+    public void singlePlatformReleaseInfo() throws Exception {
+        final PlatformProps props = newPlatformProps();
+        props.setProperty("platform.quarkus.native.builder-image", "url");
+        props.setRelease(new PlatformReleaseInfo("io.playground", "1.1", "1",
+                "io.playground:playground-bom::pom:1.1.1,io.playground:acme-bom::pom:2.2.2,io.playground:foo-bom::pom:3.3.3"));
+
+        final PlatformImports pi = new PlatformImports();
+        props.importRelease(pi);
+
+        final PlatformInfo platform = pi.getPlatform("io.playground");
+        assertNotNull(platform);
+        assertEquals("io.playground", platform.getPlatformKey());
+        final PlatformStreamInfo stream = platform.getStream("1.1");
+        assertNotNull(stream);
+        final PlatformReleaseInfo release = stream.getRelease("1");
+        assertEquals("io.playground", release.getPlatformKey());
+        assertEquals("1.1", release.getStream());
+        assertEquals("1", release.getVersion());
+        final List<AppArtifactCoords> boms = release.getBoms();
+        assertEquals(Arrays.asList(AppArtifactCoords.fromString("io.playground:playground-bom::pom:1.1.1"),
+                AppArtifactCoords.fromString("io.playground:acme-bom::pom:2.2.2"),
+                AppArtifactCoords.fromString("io.playground:foo-bom::pom:3.3.3")), boms);
+        assertEquals(1, stream.getReleases().size());
+        assertEquals(1, platform.getStreams().size());
+        assertEquals(1, pi.getPlatforms().size());
+
+        assertTrue(pi.isAligned(Collections.singletonMap("io.playground",
+                Arrays.asList(AppArtifactCoords.fromString("io.playground:playground-bom::pom:1.1.1"),
+                        AppArtifactCoords.fromString("io.playground:acme-bom::pom:2.2.2")))));
+        assertFalse(pi.isAligned(Collections.singletonMap("io.playground",
+                Arrays.asList(AppArtifactCoords.fromString("io.playground:playground-bom::pom:1.1.2"),
+                        AppArtifactCoords.fromString("io.playground:acme-bom::pom:2.2.2")))));
+    }
```

Comment:
You should start using AssertJ it's so much nicer :) (and also fails logs are nicer)